contrib/win32/win32compat/ssh-agent/agent.c

```diff
@@ -37,7 +38,7 @@
 #define BUFSIZE 5 * 1024
 
 char* sshagent_con_username;
-int sshagent_client_pid;
+HANDLE sshagent_client_primary_token;
```

Comment:
@bagajjal , Exposing the token itself as a _global variable_ is a bit riskier than exposing the client's pid.

Reply:
> @bagajjal , Exposing the token itself as a _global variable_ is a bit riskier than exposing the client's pid.

We already do this in multiple places. I don't see why it's riskier.

https://github.com/PowerShell/openssh-portable/blob/3a33ea8deed8a11245202eb442e3331a8f2793e3/contrib/win32/win32compat/spawn-ext.c#L11